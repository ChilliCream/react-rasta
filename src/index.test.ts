// Module mocks must mocked before importing modules
jest.mock("./utils/bootstrap", () => jest.fn());

import { Column, Container, Row } from ".";

describe("index", () => {
  it("should verify Column not null", () => {
    expect(Column).not.toBeNull();
  });

  it("should verify Container not null", () => {
    expect(Container).not.toBeNull();
  });

  it("should verify Row not null", () => {
    expect(Row).not.toBeNull();
  });
});
