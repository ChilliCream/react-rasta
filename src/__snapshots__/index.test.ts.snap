// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should verify the API shape 1`] = `
Object {
  "Break": [Function],
  "BreakpointMap": Object {},
  "BreakpointValue": Object {},
  "BreakpointValues": Object {},
  "Column": [Function],
  "Container": [Function],
  "Row": [Function],
  "Theme": Object {},
  "ThemeProvider": [Function],
}
`;
